src/descriptors/power_system_structs.json

```diff
@@ -101,9 +101,9 @@
         {
           "name": "ancillary_services",
           "comment": "Bids for the ancillary services",
-          "null_value": "IdDict{Any,Float64}()",
-          "data_type": "IdDict{Any,Float64}",
-          "default": "IdDict{Any,Float64}()"
+          "null_value": "IdDict{String,IS.TimeSeriesMetadata}()",
```

Comment:
If you use a String here, no need to use an `IdDict` these special dicts are useful if you use a service as a key